app/bundles/CoreBundle/EventListener/BuildJsSubscriber.php

```diff
@@ -298,7 +298,8 @@ function s4() {
 * Check if a DOM tracking pixel is present
 */
 MauticJS.checkForTrackingPixel = function() {
-    if (!/in/.test(document.readyState)) {
+    if (document.readyState != 'complete') {
```

Comment:
I suggest using the strict operator `!==`.

So this would be:
`if (document.readyState !== 'complete') {`

Reply:
Definitely agree. I usually only use strict operators, no clue why I overlooked it here.